src/main.ts

```diff
@@ -192,8 +192,8 @@ export default class ReadwisePlugin extends Plugin {
         this.notice("Syncing Readwise data");
         return this.getExportStatus(data.latest_id, buttonContext);
       } else {
-        this.handleSyncSuccess(buttonContext); // should we pass the export id to update lastSavedStatusID?
-        this.notice("Latest sync happened on different device...", false, 4, true);
+        this.handleSyncSuccess(buttonContext, "Synced", data.latest_id); // we pass the export id to update lastSavedStatusID

```

Comment:
This change will cause to show the notification only once. On subsequent syncs the notification will be "data is already up to date".